graphql/tests/projectSettings/data.json

```diff
@@ -63,6 +63,13 @@
           "field" : "sample_field"
         }]
       },
+      "container_size_definitions": [
+        {
```

Comment:
Can we please test it with an array that contains a few?

Reply:
Sure thing, added one more size to project and repo data files